test/definitions/ohi/linux/elasticsearch-linux2.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
@setamyDG the filename should be elasticsearch-rhel.json instead of elasticsearch-linux2.json